### PR TITLE
Upgrade SDK versions

### PR DESCRIPTION
**Pull request by yafuquen**
## Description
- Upgrade SDK versions
- Support notification platform `none`

## Commit message
- Upgrade SDK versions

## Testing
- [ ] Added unit tests
- [ ] Ran unit tests successfully
- [ ] Added documentation for public APIs and/or Wiki

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -47,8 +47,8 @@ android {
   defaultConfig {
     minSdkVersion 23
     targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
-    versionCode 2
-    versionName "1.1"
+    versionCode 3
+    versionName "0.3.0"
```
Comment (ferazoguerrero) [inline android/build.gradle:51]: version name should be 1.2.0? or 2.0.0

File: example/android/app/build.gradle [hunk 2]
```diff
@@ -147,8 +147,8 @@ android {
         applicationId "com.twilio.verify.reactnative.example"
         minSdkVersion rootProject.ext.minSdkVersion
         targetSdkVersion rootProject.ext.targetSdkVersion
-        versionCode 1
-        versionName "1.0"
+        versionCode 3
+        versionName "0.3.0"
```
Comment (ferazoguerrero) [inline example/android/app/build.gradle:151]: same here

**Review by ferazoguerrero — CHANGES_REQUESTED** [review]

File: example/android/app/build.gradle (hunk 2 above)
Comment (yafuquen) [reply example/android/app/build.gradle:151]: It had a wrong version, the sample app should have the same version as the SDK to be tested easily

**Review by ferazoguerrero — APPROVED** [review]

**Review by mejiagarcia — APPROVED** [review]
Looks good to me despite the forced optional values since I guess it looks like a calculated risk on the Swift code

**Pull request merged**